core/src/main/java/org/infinispan/manager/DefaultCacheManagerAdmin.java

```diff
@@ -0,0 +1,57 @@
+package org.infinispan.manager;
+
+import java.util.EnumSet;
+
+import org.infinispan.Cache;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.globalstate.GlobalConfigurationManager;
+import org.infinispan.security.AuthorizationPermission;
+import org.infinispan.security.impl.AuthorizationHelper;
+import org.infinispan.util.TimeService;
+
+/**
+ * The default implementation of {@link EmbeddedCacheManagerAdmin}
+ *
+ * @author Tristan Tarrant
+ * @since 9.2
+ */
+
+public class DefaultCacheManagerAdmin implements EmbeddedCacheManagerAdmin {
```

Comment:
I don't fully understand how security is suppose to work, but shouldn't this be internal? 
I mean, what prevents a user of doing
```
DefaultCacheManagerAdmin myHack = new DefaultCacheManagerAdmin(mySecureCache.getCacheManager(), myDummyAuth, ....)
myHack.remove(...)
```

Reply:
The constructor is package protected